public/common/TracyStackFrames.cpp

```diff
@@ -16,7 +16,7 @@ const char* s_tracyStackFrames_[] = {
     "tracy::Profiler::MemFreeCallstack(void const*, int)",
     "tracy::ScopedZone::{ctor}",
     "tracy::ScopedZone::ScopedZone(tracy::SourceLocationData const*, int, bool)",
-    "tracy::Profiler::Message",
+    "tracy::Profiler::LogString",
```

Comment:
Old function must also be here, as it is used for filtering in the UI and you can have traces from before this change.